tensorflow_addons/custom_ops/image/BUILD

```diff
@@ -27,6 +29,7 @@ py_library(
         "__init__.py",
         "python/__init__.py",
         "python/transform.py",
+        "python/distance_transform.py",
```

Comment:
```
"python/distance_transform.py",
"python/transform.py",
```

Reply:
@Squadrick Hi, could you merge the latest master branch, and run `make code-format` in the docker container? See https://github.com/tensorflow/addons/blob/master/CONTRIBUTING.md#coding-style for more details :-)

Reply:
That is good to know :)